### PR TITLE
net-irc/quasselgrep version bump (0_p20190211|9999), added python 3.(…

### PR DESCRIPTION
**Pull request by ephemer0l**
…5,6,7) support

### CODE REVIEW

File: net-irc/quasselgrep/quasselgrep-0_p20190211.ebuild [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+DESCRIPTION="Tool for searching quassel logs from the commandline"
+HOMEPAGE="https://github.com/fish-face/quasselgrep"
+
+if [[ "${PV}" == "9999" ]] ; then
+	EGIT_REPO_URI="https://github.com/65278/quasselgrep.git"
+	inherit git-r3
+	KEYWORDS=""
+else
+	MY_COMMIT=9b6b0bc1252daa6e574363d87d04eebd981215a5
+	SRC_URI="https://github.com/fish-face/"${PN}"/tarball/"${MY_COMMIT}" -> "${P}".tar.gz"
```
Comment (mgorny) [inline net-irc/quasselgrep/quasselgrep-0_p20190211.ebuild]: The mid-`"` make no sense here.

File: net-irc/quasselgrep/quasselgrep-0_p20190211.ebuild [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+	S=""${WORKDIR}"/fish-face-"${PN}"-"${MY_COMMIT:0:7}""
+	KEYWORDS="~amd64 ~x86"
+#else
```
Comment (mgorny) [inline net-irc/quasselgrep/quasselgrep-0_p20190211.ebuild]: Why are you leaving commented out code in?

File: net-irc/quasselgrep/quasselgrep-0_p20190211.ebuild [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+IUSE="sqlite postgres"
+
+DEPEND="${RDEPEND}"
+RDEPEND="sqlite? ( "dev-lang/python[sqlite]" )
+	dev-python/python-dateutil[${PYTHON_USEDEP}]
+	dev-python/pycryptodome[${PYTHON_USEDEP}]
+	dev-python/future[${PYTHON_USEDEP}]
+	postgres? ( dev-python/psycopg:2[${PYTHON_USEDEP}] )"
+
+#src_configure() {
```
Comment (mgorny) [inline net-irc/quasselgrep/quasselgrep-0_p20190211.ebuild]: Same.

File: net-irc/quasselgrep/quasselgrep-0_p20190211.ebuild [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+#	SRC_URI="https://github.com/fish-face/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+#	KEYWORDS="~amd64 ~x86"
+fi
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="sqlite postgres"
+
+DEPEND="${RDEPEND}"
+RDEPEND="sqlite? ( "dev-lang/python[sqlite]" )
```
Comment (mgorny) [inline net-irc/quasselgrep/quasselgrep-0_p20190211.ebuild]: Mid-quoting doesn't work. Also please sort.

File: net-irc/quasselgrep/quasselgrep-0_p20190211.ebuild [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+#else
+#	SRC_URI="https://github.com/fish-face/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+#	KEYWORDS="~amd64 ~x86"
+fi
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="sqlite postgres"
+
+DEPEND="${RDEPEND}"
```
Comment (mgorny) [inline net-irc/quasselgrep/quasselgrep-0_p20190211.ebuild]: RDEPEND is not set at this point yet.

**Comment by mgorny** [comment]
Ping.

**Comment by juippis** [comment]
fc7a3a5ed3576f8793ca4b2baa959745142166f5

**Pull request closed without merging**